src/index.js

```diff
@@ -1,18 +1,112 @@
 import './style.css';
 
+const refreshButton = document.querySelector('#refresh');
+
+const { bootstrap } = window;
+
 const form = document.forms[0];
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
 
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
-  const name = form.elements[0];
-  const score = form.elements[1];
+async function fetchData(endpoint, data, method) {
+  const result = await fetch(URL + endpoint, {
+    method,
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json; Charset=UTF-8',
+    },
+  });
+
+  return result;
+}
+async function createGame() {
+  const endpoint = 'games/';
+  const data = { name: 'Awesome Game' };
+  const result = await fetchData(endpoint, data, 'POST');
+  const json = await result.json();
+  const id = json.result;
+  const actualId = id.split('Game with ID: ')[1].split(' added.')[0];
+  return actualId;
+}
+
+function saveGameIdLocally(id) {
+  localStorage.setItem('gameId', JSON.stringify(id));
+}
+
+function retreiveGameId() {
+  const id = JSON.parse(localStorage.getItem('gameId'));
+  return id;
+}
+
+function createScoreElement(userName, score) {
```

Comment:
- [ ] I would recommend using `arrow functions` instead of function keyword in all the places.